doc/api/lxml.tests.rst

```diff
@@ -0,0 +1,282 @@
+lxml.tests
```

Comment:
I think tests were previously only included by accident. I think it's useless to add them here.

Reply:
Removed tests.